core/src/saros/session/internal/ActivitySequencer.java

```diff
@@ -359,7 +359,7 @@ public void sendActivity(List<User> recipients, final IActivity activity) {
           new Runnable() {
             @Override
             public void run() {
-              sarosSession.exec(Collections.singletonList(activity));
+              sarosSession.exec(user.getJID(), Collections.singletonList(activity));
```

Comment:
`getJID` is deprecated. However, I found no alternative method (as e.g. `getXMPPContact`/`getContact`).
@stefaus How would you get the Contact/JID?

Reply:
At the moment there is no proper replacement for everything where you need the contact plus the correct resource endpoint. `XMPPContact` can return the current best available `JID`, but this could change at any time.

the `user.getJID()` deprecation is really old, but until we have something better working this PR looks good to me